src/main/webapp/help-globalConfig-personalNotify.html

```diff
@@ -0,0 +1,3 @@
+<div>
+    <p>When checked, build notifications will be posted to committers as private messages.</p>
```

Comment:
Since the whole personal notification idea is based around committer email, i'm guessing this won't work for users that logged in to Zulip via Github or Google?
If that's the case we should definitely let user know in the help / docs / readme.

Reply:
That's true. I'll document that.